ansible/roles/neutron/templates/neutron-server.json.j2

```diff
@@ -25,6 +25,12 @@
             "owner": "neutron",
             "perm": "0600"
         },
+        {
+            "source": "{{ container_config_directory }}/ssh/",
+            "dest": "/etc/neutron/ssh",
```

Comment:
I think you need `"optional": true,` b/c it's possible the tasks that write these files will be skipped.